src/App.php

```diff
@@ -2006,4 +2006,20 @@ public function internalRedirect($toUrl = '', $ssl = false)
 		$redirectTo = $this->getBaseURL($ssl) . '/' . ltrim($toUrl, '/');
 		Core\System::externalRedirect($redirectTo);
 	}
+
+	/**
+	 * Redirects to another URL in case
```

Comment:
in case?

Reply:
Ah damn, bad translation of a german meaning ^^